apps/freeablo/faworld/missile/missilegraphic.cpp

```diff
@@ -19,16 +24,28 @@ namespace FAWorld::Missile
         mMainGraphicPath = loader.load<std::string>();
         mSingleFrame = (mSingleFrame = loader.load<int32_t>()) == -1 ? std::nullopt : mSingleFrame;
         mAnimationPlayer = FARender::AnimationPlayer(loader);
+        auto levelIndex = loader.load<int32_t>();
+        auto world = loader.currentlyLoadingWorld;
+        loader.addFunctionToRunAtEnd([this, world, levelIndex]() {
+            mLevel = world->getLevel(levelIndex);
+            mLevel->mMissileGraphics.push_back(this);
```

Comment:
It's not currently a probem, since we only use this list for rendering,  but the order of the list will not be the same when loading a game as it was before saving. If we ever use the list for any kind of game logic, this could be a problem. Maybe add a comment there that warns not to use it for game logic. If we ever do need them, then we can make all of the inserts sort by some serialised unique id or something similar.